### DISCUSSION TITLE
Can't enter private key after the "forge create SimpleSotrage --interactive" command.

### DISCUSSION THREAD

**Discussion by Hepcat93** [Cyfrin/foundry-full-course-cu#2906] [Q&A] [answered]
Hi everyone. I'm currently studying the Foundry Fundamentials course, Section 1: Foundry Simple Storage, Lesson 15 "Deploy a smart contract locally using Forge".

My problem is that when I run the "forge create SimpleSotrage --interactive" command in VS Code terminal, I'm offered to enter a private key, but I can't enter anything like Patrick does in the video. The command line doesn't respond to any key except the Enter key. Of course it's easy to circumvent with the "forge create SimpleStorage --private-key" command, but I don't want to clean the history every time :D

Can this be somehow connected with the fact that I'm using WSL/Ubuntu for terminal in VS Code? Not Bash. Previous lessons with installing and setting up Foundry confused me little, so this is how it worked out for me.

**Comment by LordAdhaar** [comment, marked as answer]
For security purposes passwords cannot be seen in the terminal. Even if you are typing you will not seen anything in the terminal. Also you can paste the private key instead of typing it out.

**Reply by Hepcat93** [reply]
Oh, how confusing, I faced this thing when was installing that Ubuntu distro... Thanks! I was unattentive :(

**Reply by LordAdhaar** [reply]
No issue. Later in scripts you will need to use `--interactives 1` instead of `--interactive`. Just giving you a heads up, so you don't end up spending a lot of time to solve that issue like I did.

**Reply by EngrPips** [reply]
Nice of you.

**Reply by Hepcat93** [reply]
Thanks man :D

**Comment by heheboii11** [comment]
You can use ```cast wallet import --interactive privatekey```, (after interactive you have to give a name, you can give any but here i have saved it as ```privatekey```, you can give any name) after you use this prompt in the terminal it will ask you to provide a private key, and choose a password for encrypting it and it will be stored. you can use this private-key for further deployments.

**Reply by Hepcat93** [reply]
Also thanks for sharing this command, however I wonder if it will be safe to use it with real accounts later... But comfortable while doing things in test networks - for sure!

**Reply by EngrPips** [reply]
I think it is.

**Reply by heheboii11** [reply]
It is safe as far I am aware but as always never use your degen account or account that you hold your crypto, just create a new account and use it on testnets!! Stay SAFU

**Comment by EngrPips** [comment]
Great heads-up.
